server/models/favorite.model.js

```diff
@@ -0,0 +1,70 @@
+import Promise from 'bluebird';
+import mongoose from 'mongoose';
+import httpStatus from 'http-status';
+import APIError from '../helpers/APIError';
+
+/**
+ * Favorite Schema
+ * @property {String} userId - The id of the user.
+ * @property {ObjectId} postId - The id of the post.
+ * @property {Boolean} active - The active state of the favorite.
+ */
+const FavoriteSchema = new mongoose.Schema({
+  userId: String,
```

Comment:
This should be an objectID reference to users

Reply:
Like this: `userId: { type: mongoose.Schema.Types.ObjectId, ref: 'User' },`?